### PR TITLE
Use the new FxA Client 0.5.0 AAR

### LINKED ISSUE #815: Fetch `org.mozilla.fxa_client:fxa_client` for FxA client native libraries
Right now, we use the TC index to pull a ZIP of FxA client native libraries.  As of https://github.com/mozilla/application-services/pull/248 we should be publishing a well-formed AAR with these dependencies to bintray (and jcenter, eventually) and no longer publishing the ZIP file.  This ticket tracks updating and testing the update.

/cc @vladikoff

### PR DESCRIPTION
**Pull request by vladikoff**
Fixes #815 

cc @ncalexan

### CODE REVIEW

File: components/service/firefox-accounts/build.gradle [hunk 1]
```diff
@@ -28,11 +28,8 @@ android {
 }
 
 repositories {
-    ivy {
-        url 'https://index.taskcluster.net/v1/task/'
-        layout 'pattern', {
-            artifact '/project.fxaclient.builds.[revision]/artifacts/public/bin/[organisation]/[module]_[revision].[ext]'
-        }
+    maven {
```
Comment (vladikoff) [inline components/service/firefox-accounts/build.gradle]: we can remove it once the thing is on jcenter

File: components/service/firefox-accounts/build.gradle [hunk 2]
```diff
@@ -73,12 +70,6 @@ dependencies {
 
 ext.jniOutputDir = "src/main/jniLibs"
```
Comment (csadilek) [inline components/service/firefox-accounts/build.gradle]: Should be able to remove this line as well now!

**Comment by vladikoff** [comment]
Rebased!

**Comment by vladikoff** [comment]
> @vladikoff did we request jcenter sync for fxa_client already? If so, it might be worth waiting...

No, it got rejected, but we are trying again

**Comment by csadilek** [comment]
@vladikoff looks like we landed on jcenter: https://jcenter.bintray.com/org/mozilla/fxa_client/

Do you have time to update this? If not, I will merge an send a follow-up. We'll release in the next few hours.

**Comment by vladikoff** [comment]
Please go ahead! Thanks!

**Comment by csadilek** [comment]
Closing this and re-submitting with Nick's repo removed.

**Pull request closed without merging**